components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1603,6 +1603,25 @@ _resolve_dependencies() {
   _validate_deps
 }
 
+function do_automatic() {
+  do_default_automatic "$1"
+  return $?
+}
+
+# Loads specified automatic to extend plan DSL.
+function do_default_automatic() {
+  local automatic="${automatic:-$1}"
+
+  if [[ -n "$automatic" ]]; then
+    build_line "Loading Automatic $automatic"
+    if source "$(pkg_path_for $automatic)/bin/automatic.sh"; then
+      build_line "Automatic loaded"
+    else
+      exit_with "Failed to load Automatic $automatic" $ret
```

Comment:
Oops. That's leftover from a small refactor I did at one point and forgot to fix it. Thanks!